ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -769,6 +794,46 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-period':
+          test_name: windows-e2e-userspace
+          node: 'antrea-windows-testbed'
+          description: 'This is for running e2e tests in userspace mode for testbed with conatinerd runtime.'
+          builders:
+            - builder-e2e-win:
+                win_image_node: '{antrea_win_image_node_name}'
```

Comment:
`win_image_node` can be defined in `govc.env` now, so there's no need to include it in the job YAML anymore. Additionally, we want to avoid sharing the same image node across multiple clusters. Please remove it from the other jobs listed in projects-lab.yaml and macros.yaml as well.